src/fty_common_db_asset.cc

```diff
@@ -433,6 +433,67 @@ int select_assets_by_container(tntdb::Connection& conn, uint32_t element_id, std
             }
         }
 
+        // if configured option is defined
+        if (!configured.empty()) {
```

Comment:
test the value.  configured == "yes" or "no" instead

Reply:
Take into account